cartridges/int_algolia/cartridge/scripts/algolia/model/algoliaLocalizedProduct.js

```diff
@@ -321,30 +321,19 @@ var aggregatedValueHandlers = {
  * AlgoliaLocalizedProduct class that represents a localized algoliaProduct ready to be indexed
  * @param {dw.order.Product} product - Product
  * @param {string} locale - The requested locale
- * @param {Array} [fieldListOverride] (optional) if supplied, it overrides the regular list of attributes to be sent (default + customFields)
+ * @param {Array} [fieldList] list of attributes to be fetched
```

Comment:
Since we agreed to change the public name of the preference, we might as well change the name of the variables and parameters and all variables/parameters containing "field" as well for consistency's sake. This one could be `attributeList`.